### ISSUE TITLE
Colour contrast of buttons

### ISSUE THREAD

**Issue by jamie-benoit** [nsgov/patternlab#10]
Geoff did an accessibility review  of one of our service and he noted this:

> For the buttons, there isn’t enough colour contrast between the colour of the buttons in the normal state and in the :hover state (1.54:1, should be 3:1 or preferably even 4.5:1). The colour contrast should be increased. Changing it to be the same as the :focus state - which has the added yellow outline - would help.

The service is the colour of button from this pattern library. I was hoping that we could look at colour contrast in the pattern library and not in the service.

**Comment by hypnokayte** [comment]
👋 Hey Jamie! I've left NSGov, but @nordlund should be able to help you out with this 😄
